content/remote-teaching/keeping-pupils-motivated-and-engaged.gs

```diff
@@ -1,8 +1,14 @@
 ---
-title: Keeping pupils motivated and engaged
+title: Keeping pupils motivated and engaged at a distance
 lead_paragraph: Schools have been using various methods to ensure that they are supporting pupils’ wellbeing and providing pastoral care.
 date: "5 May 2020"
-related_content:
+- name: Supporting wellbeing at a distance
```

Comment:
I think you're missing a `related_content:` key here

Reply:
Thank you, solved now.